fibers.scm

```diff
@@ -120,11 +120,21 @@
       (unless init
         (error "run-fibers requires initial fiber thunk when creating sched"))
       (spawn-fiber (lambda ()
-                     (call-with-values init
-                       (lambda vals (atomic-box-set! ret vals)))
-                     ;; Could be that this fiber was migrated away.
-                     ;; Make sure to wake up the main scheduler.
-                     (spawn-fiber (lambda () #t) scheduler))
+                     (dynamic-wind
```

Comment:
Spacing seems wonky, though maybe that's just a different tabs / space convention (does someone know the conventions used in Fibers?).